hack/build/config.sh

```diff
@@ -47,9 +47,9 @@ OPERATOR_IMAGE_NAME=${OPERATOR_IMAGE_NAME:-cdi-operator}
 DOCKER_TAG=${DOCKER_TAG:-latest}
 VERBOSITY=${VERBOSITY:-1}
 PULL_POLICY=${PULL_POLICY:-IfNotPresent}
-NAMESPACE=${NAMESPACE:-kube-system}
+NAMESPACE=${NAMESPACE:-cdi}
 
-KUBERNETES_IMAGE="k8s-1.11.0@sha256:3412f158ecad53543c9b0aa8468db84dd043f01832a66f0db90327b7dc36a8e8"
+KUBERNETES_IMAGE="k8s-1.11.0@sha256:e02ec414c7673a3644b5ba742b550a124c9195eaacb280151406bf9e8201a95f"
```

Comment:
Yeah is there something in that image you require? if so we can make a PR to update it outside of this PR.

Reply:
I wouldn't call it sneaky, I am just wondering what the purpose of it is in regards to this PR. We usually keep image changes in their own PR.

Reply:
Purpose was just good housekeeping.  Would have updated the openshift image if it wasn't already up to date.  Since it is same k8s version I figured it was nbd and did not require a new pr.

Reply:
Okay, well since this PR will probably get backported it makes sense to make as few changes as possible (even though this particular one doesn't matter at all).